app/src/main/java/com/keylesspalace/tusky/components/preference/AccountPreferencesFragment.kt

```diff
@@ -84,7 +84,7 @@ class AccountPreferencesFragment : PreferenceFragmentCompat(), Injectable {
                     colorInt = MaterialColors.getColor(context, R.attr.iconColor, Color.BLACK)
                 }
                 setOnPreferenceClickListener {
-                    openNotificationPrefs()
+                    openNotificationSystemPrefs()
```

Comment:
Changed the name to make it more clear that this leaves the "scope of the app code".